images/kube-vip/README.md

```diff
@@ -0,0 +1,124 @@
+<!--monopod:start-->
+# kube-vip
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/kube-vip` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/kube-vip/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+Kubernetes Control Plane Virtual IP and Load-Balancer
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/kube-vip:latest
+```
+<!--getting:end-->
+
+<!--body:start-->
+
+## Usage
+
```

Comment:
Can we include the helm chart example here? this section should show how to use with the supported helm charts (https://github.com/kube-vip/helm-charts/blob/main/charts/kube-vip/values.yaml).

I see we've one in the helm automated test so we should at least link this chart here, with an example of how to override the image.

If we're not able to automate everything from a testing perspective, we should move the manual test steps (i.e here), to a TESTING.md file